xclim/testing/tests/test_sdba/test_adjustment.py

```diff
@@ -477,6 +477,39 @@ def test_mon_U(self, mon_series, series, mon_triangular, kind, name):
         # Test predict
         np.testing.assert_array_almost_equal(p, ref, 2)
 
+    @pytest.mark.parametrize("use_dask", [True, False])
+    def test_add_dims(self, use_dask):
+        if use_dask:
+            chunks = {"location": -1}
+        else:
+            chunks = None
+        ref = (
+            open_dataset(
+                "sdba/ahccd_1950-2013.nc", chunks=chunks, drop_variables=["lat", "lon"]
+            )
+            .sel(time=slice("1981", "2010"))
+            .tasmax
+        )
+        ref = convert_units_to(ref, "K")
+        ref = ref.isel(location=1, drop=True).expand_dims(location=["Amos"])
+
+        dsim = open_dataset(
+            "sdba/CanESM2_1950-2100.nc", chunks=chunks, drop_variables=["lat", "lon"]
+        ).tasmax
+        hist = dsim.sel(time=slice("1981", "2010"))
+        sim = dsim.sel(time=slice("2041", "2070"))
+
+        # With add_dims, "does it run" test
+        group = Grouper("time.dayofyear", window=5, add_dims=["location"])
+        EQM = EmpiricalQuantileMapping.train(ref, hist, group=group)
+        EQM.adjust(sim).load()
+
+        # Without, sanity test.
+        group = Grouper("time.dayofyear", window=5)
```

Comment:
Is it a breaking change now that window can only be passed inside Grouper? @aulemahal

Reply:
I.e. shouldn’t that be also mentioned in History and the docstrings

Reply:
Did this break? On my side, EQM works by passing them as kwargs.
In theory, both ways should work as whenever you can pass `group` as a string, you should also be able to pass `window` and `add_dims`  as extra kwargs. In my tests and code, I simply prefer to create the `Grouper` object ahead and reuse it, but we made the code so both should be possible.

Reply:
I had to pass window in Group now which worked before outside group but I also used add_dims, so it might be due to the combination

Reply:
Sorry about that. With the current master, I was able to create EQM, DQM, QDM, LOCI and Scaling with:
```
adj = ADJ.train(ref, hist, group='time.dayofyear', window=10, add_dims=['location'], **kwargs)
ad.group
# Grouper(add_dims=['location'], name='time.dayofyear', window=10, interp=False)
```
The kwargs parsing stuff happens before the actual training is called (even before the Adjustment object is created). 

What does your failing code look like?

Reply:
Oh I realize now that what might have changed is the behaviour is `group` is not given! 
I see this is a breaking change and we should have noted it, sorry!

But, `window > 1` has no meaning if `group='time'`, no?  I mean it will work and results will be different than without, but it doesn't do anything statistically significative. For `add_dims`, though, this is a a clear loss.

Reply:
I think I was confused about how to use Group instead of group and where to put the kwargs like window or add_dims. I wrongly thought that they need to be in Group but that isn’t necessary.